doc/sphinx/how-to/how-to-test-mir-for-a-release.md

```diff
@@ -0,0 +1,174 @@
+# How to test Mir for a release
+The following document details the criteria that must be met before we can
+publish an official release of Mir. When adding to this test plan, please
+do not fail to consider how your additions may be automated in the future, as
+this is the end goal of our release testing.
+
+Note that all of these tests must be run _after_ a release branch is created
+in the Mir repository, such that the right repositories are updated beforehand.
+
+## Setup
+1. Install mir from the release candidate PPA
+    ```
+    sudo add-apt-repository --update ppa:mir-team/rc
+    sudo apt install mir-demos mir-platform-graphics-virtual mir-graphics-drivers-desktop mir-test-tools
+    ```
+2. Install the `mir-test-tools` from the `22/beta` _or_ `24/beta` track (depending
+   on which you are trying to test):
+    ```
+    sudo snap refresh
+    sudo snap install mir-test-tools --channel=24/beta
+    # ... or ...
+    sudo snap install mir-test-tools --channel=22/beta
+    ```
+3. Install `miriway` from the latest `beta` track:
+    ```
+    sudo snap refresh
+    sudo snap install miriway --classic --chanel=latest/beta
+    ```
+
+## Tests
+Manual tests must be performed across a combination of different display
+platforms and different Ubuntu releases. It is important also that we test both
+the `snap` and the `non-snap` version for each platform.
+
+|                                | core24 snap | 24.04 LTS | 24.10 |
+|--------------------------------|-------------|----------|------------|
+| gbm-kms                        |             |          |            |
+| eglstream-kms                  |             |          |            |
+| eglstream-kms + gbm-kms hybrid |             |          |            |
+| x11                        |             |          |            |
+| wayland                    |             |          |            |
+| virtual                        |             |          |            |
```

Comment:
I don't think the "core24 snap" column works here, and it is definitely not a reorg of the existing test plan.

We know, for instance, that the Miriway snap currently cannot work on `eglstream-kms`, but that the Frame snap can. OTOH Frame does not support `virtual` nor `wayland`, while Miriway does.

Reply:
Yup, I think having different documents for each Mir-based compositor is the way to go in this instance.